src/utils/bitSet.ts

```diff
@@ -61,3 +61,46 @@ export const toObject = ( bitMask: TBitSetMask = {}, value = 0 ): TBooleanObject
 
     return result;
 };
+
+/**
+ * Extract number of bits from number from given position.
+ *
+ * @example
+ * ```js
+ * const value = 7; // 0b111
+ *
+ * console.log(extractBitsFromNumber(value, 2, 1));
+ * // 3
+ * // i.e. 0b011
+ * ```
+ */
+export const extractBitsFromNumber = (
```

Comment:
extractBitsFromPosition? slice? extractBits?

Reply:
renamed to `extractBits`, `setBitsToNumber` -> `fillBits`